### PR TITLE
Style changes, clarify websocket pricing for DO

### PR DESCRIPTION
**Pull request by greg-mckeon**

### CODE REVIEW

**Review by deadlypants1973 — APPROVED** [review]

File: products/workers/src/content/runtime-apis/durable-objects.md [hunk 1]
```diff
@@ -421,5 +421,5 @@ Any uncaught exceptions thrown by the Durable Object's `fetch()` handler are pro
 
 The Cloudflare API supports retrieving a list of Durable Objects within a namespace and a list of namespaces associated with an account.
 
-The API for listing namespaces is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces
-The API for listing objects in a namespace is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects
+The API for listing namespaces is available [here](https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces).
+The API for listing objects in a namespace is available [here](https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects).
```
Comment (koeninger) [inline products/workers/src/content/runtime-apis/durable-objects.md]: same comment as original pr - this shouldn't be a hyperlink, the url is the information we're trying to convey, and isn't useful until someone replaces their account id / namespace id in it

File: products/workers/src/content/runtime-apis/durable-objects.md [hunk 2]
```diff
@@ -421,5 +421,5 @@ Any uncaught exceptions thrown by the Durable Object's `fetch()` handler are pro
 
 The Cloudflare API supports retrieving a list of Durable Objects within a namespace and a list of namespaces associated with an account.
 
-The API for listing namespaces is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces
-The API for listing objects in a namespace is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects
+The API for listing namespaces is available [here](https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces).
```
Comment (deadlypants1973) [inline products/workers/src/content/runtime-apis/durable-objects.md]:
```suggestion
The API for listing namespaces is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces.
```

File: products/workers/src/content/runtime-apis/durable-objects.md (hunk 1 above)
Comment (deadlypants1973) [inline products/workers/src/content/runtime-apis/durable-objects.md]:
```suggestion
The API for listing objects in a namespace is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects.
```

Comment (koeninger) [reply products/workers/src/content/runtime-apis/durable-objects.md]: v4 api docs for durable objects are now live at

https://api.cloudflare.com/#durable-objects-namespace-list-namespaces
and
https://api.cloudflare.com/#durable-objects-namespace-list-objects

if we want to link there instead

File: products/workers/src/content/runtime-apis/durable-objects.md [hunk 3]
```diff
@@ -421,5 +421,5 @@ Any uncaught exceptions thrown by the Durable Object's `fetch()` handler are pro
 
 The Cloudflare API supports retrieving a list of Durable Objects within a namespace and a list of namespaces associated with an account.
 
-The API for listing namespaces is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces
-The API for listing objects in a namespace is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects
+The API for listing namespaces is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces.
+The API for listing objects in a namespace is available at https://api.cloudflare.com/client/v4/accounts/$YOUR_ACCOUNT_ID/workers/durable_objects/namespaces/$NAMESPACE_ID/objects.
```
Comment (koeninger) [inline products/workers/src/content/runtime-apis/durable-objects.md:425]: This section is correct in production as of https://github.com/cloudflare/cloudflare-docs/pull/2824 and shouldn't be changed back here.

**Review by koeninger — CHANGES_REQUESTED** [review]

**Pull request closed without merging**